js/classes/entities/foreground/dialog/decision_bubble.js

```diff
@@ -0,0 +1,106 @@
+class DecisionBubble {
+  
+  constructor(questioner, prompt, choices) {
+    /** The name of the person asking Chad the question. */
+    this.questioner = questioner;
+    /** A brief (one line max!) reminder to the player about the decision they're about to make. */
+    this.prompt = prompt;
+    /** Up to four BRIEF indicators of the options Chad has. */
+    this.choices = choices;
+    /** The index of the selected (highlighted) option. */
+    this.selected = 0;
+  };
+
+  update() {
+    if (GAME.user.choiceUp) {
+      // User is pushing up to make a choice at an earlier index.
+      this.selected--;
+
+      if (this.selected < 0) {
+        // Did we move back from the first option? Choose the last then!
+        this.selected = this.choices.length - 1;
+      }
+
+    }
+
+    if (GAME.user.choiceDown) {
+      // User is pushing down to make a choice at a later index.
+      this.selected++;
+      if (this.selected >= this.choices.length) {
+        // Did we move forward from the last option? Back to the beginning!
+        this.selected = 0;
+      }
+    }
+  };
+
+  draw() {
+    // (1)  Draw the dialog bubble.
+    const dbPos = new Vector( // position of decision bubble on canvas.
```

Comment:
Not sure if this will be confusing using "dbPos" when we intend to use a database in the future for saving. I feel like it could be confused for db as in database as I originally read it.

Reply:
I don't disagree that it's similar, but this is a local variable in the DialogBubble (and DecisionBubble) classes - will never have anything to do with databases.